lib/galaxy/webapps/galaxy/api/workflows.py

```diff
@@ -43,6 +43,7 @@
     MissingToolsException,
     RefactorRequest,
     WorkflowCreateOptions,
+    WorkflowsManager,
```

Comment:
```suggestion
```

Reply:
sorry, still fighting with all the IDE setups for this project now :D